src/LandIce/problems/LandIce_StokesFOThermoCoupled.hpp

```diff
@@ -159,34 +153,59 @@ constructEvaluators (PHX::FieldManager<PHAL::AlbanyTraits>& fm0,
       p->set<std::string>("Updated Bed Topography Name", bed_topography_name);
       p->set<Teuchos::ParameterList*>("Physical Parameter List", &params->sublist("LandIce Physical Parameters"));
 
-      ev = Teuchos::rcp(new UpdateZCoordinateMovingBed<EvalT,PHAL::AlbanyTraits>(*p, dl));
+      ev = Teuchos::rcp(new LandIce::UpdateZCoordinateMovingBed<EvalT,PHAL::AlbanyTraits>(*p, dl));
```

Comment:
Curious: were these namespace specs necessary? Should compile even without, right?

Reply:
no, they are not necessary. I don't recall why I added them.